grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -704,7 +704,7 @@ class OrderService {
                     }
 
                     BigDecimal parsedUnitPrice = CSVUtils.parseNumber(unitPrice, "unitPrice")
-                    if (orderItem.hasInvoices && orderItem.unitPrice != parsedUnitPrice) {
+                    if (orderItem.hasInvoices && !orderItem.hasPrepaymentInvoice && orderItem.unitPrice != parsedUnitPrice) {
```

Comment:
What if there will be a prepayment invoice and a final invoice? Perhaps it would be better to check if `orderItem.hasRegularInvoice`